package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gpii-pouch",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "test": "node src/tests/all-tests.js"
+  },
+  "dependencies": {
+    "express":         "4.10.5",
+    "express-pouchdb": "0.14.0",
+    "infusion":        "git://github.com/fluid-project/infusion.git#df5f8cfabf815a4086b778e73125c3c952dda4ec",
+    "memdown":         "1.0.0",
+    "pouchdb":         "3.3.1",
+    "underscore-node": "0.1.2"
+  },
+  "devDependencies": {
+    "gpii-express":         "git://github.com/the-t-in-rtf/gpii-express#dbb1f38368fe9534fcd1961c1095ba0661e3d877",
```

Comment:
I assume you meant updating the `gpii.express` dependency to use the GPII repository rather than my own.